modular_citadel/code/modules/reagents/chemistry/reagents/MKUltra.dm

```diff
@@ -266,55 +266,20 @@ Creating a chem with a low purity will make you permanently fall in love with so
 //Creates a gas cloud when the reaction blows up, causing everyone in it to fall in love with someone/something while it's in their system.
 /datum/reagent/fermi/enthrallExplo//Created in a gas cloud when it explodes
 	name = "Gaseous MKUltra"
-	description = "A forbidden deep red gas that overwhelms a foreign body, causing the person they next lay their eyes on to become more interesting. Studies have shown that people are 66% more likely to make friends with this in the air. Produced when MKUltra explodes."
+	description = "A deep red gas that causes the mind to feel absentminded for as long as it is in their system. Produced when MKUltra explodes."
```

Comment:
"the mind to feel absentminded"? wording sounds a little odd to me and I don't think absentminded is one word either

Reply:
Maybe what I have next sounds better.